pkg/generators/forwarding/fluentd/output_conf_loki_test.go

```diff
@@ -0,0 +1,82 @@
+package fluentd
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	test "github.com/openshift/cluster-logging-operator/test"
+)
+
+var _ = Describe("Generating external loki server output store config block", func() {
+	var (
+		err       error
+		outputs   []v1alpha1.OutputSpec
+		generator *ConfigGenerator
+	)
+	BeforeEach(func() {
+		generator, err = NewConfigGenerator(false, false)
+		Expect(err).To(BeNil())
+	})
+
+	Context("for a loki endpoint", func() {
+		lokiConf := `<label @LOKI_RECEIVER>
+        <match **>
+            @type loki
+            url https://logs-us-west1.grafana.net
```

Comment:
This cluster will be removed at some point.

Reply:
This is just a unit test for the config generator. It is now using the cluster at all. Probably I have copied the URL from the plugin's readme to right the test in the first place.